### ISSUE TITLE
--log-file not use file to write logs, use console stout.

### ISSUE THREAD

**Issue by bizlevel** [pypiserver/pypiserver#245]
on fresh debian 9 host i run pipy as systemctl service with string:

`ExecStart=/usr/local/bin/pypi-server -p 8080 -a update,download --log-file /var/log/pypiserver.log -P /etc/nginx/.htpasswd /var/www/pypi`

Before i create **/var/log/pypiserver.log** Because i got error for this file.
But when i run this:
```
sudo touch /var/log/pypiserver.log
sudo chmod 666 /var/log/pypiserver.log
```
and start service, i not get output to that file - i got all echo to /var/syslog !
Some string from syslog:
```
Feb 14 15:18:24 ip-172-31-33-162 pypi-server[1992]: 128.0.xx.xx - - [14/Feb/2019 15:18:24] "GET / HTTP/1.1" 200 857
Feb 14 15:30:25 ip-172-31-33-162 pypi-server[1992]: 128.0.xx.xx - - [14/Feb/2019 15:30:25] "GET / HTTP/1.1" 200 857
Feb 14 15:30:28 ip-172-31-33-162 pypi-server[1992]: 128.0.xx.xx - - [14/Feb/2019 15:30:28] "GET /packages/ HTTP/1.1" 200 173
```

Why? Why ask permission for write to log-file, but not write to them? )

**Comment by mplanchard** [comment]
I believe that those are bottle's logs, which we're just updating with `pypi-server` to make the source clear. If you set your verbosity higher with `-vvv`, do you see output in the logfile?

**Comment by bizlevel** [comment]
Yes, if i use "**-v**" or "**-vvv**" - all is fine, all is correct. But withouth "**-v**" no logs, even if we use "**--log-file**"
And one question - what is minimal verbosity? If i will use "**-v**" it is not mininmal?
Thanks.

**Comment by mplanchard** [comment]
1, 2 and 3 `-v`s correspond to the `WARNING`, `INFO`, and `DEBUG` logging levels in Python, respectively. There are only three places in `pypiserver`'s code that emit log messages at the `WARNING` level. The situations that trigger warning logs are currently:

* someone tries to upload a file that already exists, without the overwrite option specified
* an `update` request is submitted with an unsupported action
* the `welcome.html` file for the index cannot be loaded
